Add behavior to allow custom claims validation (#175) (#179)
arcus-azure/arcus.webapi: docs/features/security/auth/jwt.md

        });
    }
}
```

### Custom Claim validation

> It allows validating not only not claims that are added to the authorization in the JWT token, but any type of custom claim that needs to be verified

Reviewer: ```suggestion
It allows validating not only on the audience claim in the JWT token, but any type of custom claim that needs to be verified
```

Double negation. I guess you meant this :)

Author: done

```csharp
public class Startup
{

>     public void ConfigureServices(IServiceCollection services)

Reviewer 2: Based on https://github.com/arcus-azure/arcus.webapi/pull/179/files#diff-48f283b1ac9786c91d89d9bb04fcc05cf8d48411eb1086840a67a7de9dcf98cfR33 this looks outdated, is that correct?

Author: this is the latest version, nothing has changed here
